eng/common/scripts/Generate-DocIndex.ps1

```diff
@@ -0,0 +1,147 @@
+# Generates an index page for cataloging different versions of the Docs
+[CmdletBinding()]
+Param (
+    $RepoRoot,
+    $DocGenDir,
+    $lang = "c", # Take c for example. should be empty string
+    $packageNameRegex = "",
+    $packageNameReplacement = "",
+    $blobPath = "docs" # This is a setting for c. should be empty string for all other repo.
+)
+. (Join-Path $PSScriptRoot ../common/scripts/Package-Properties.ps1)
+
+# A mapping is used to fill in 1. Title of home page, 2. The azure storage link
+$LangMapping = @{
```

Comment:
We shouldn't need this mapping. If we need this information we should put it into the Language-Settings.ps1 file.